### ISSUE TITLE
[core][game_events] Bash´ir landing - problem if server crashed/restarts in middle of ongoing event

### ISSUE THREAD

**Issue by Evolvee** [Atlantiss/NetherwingBugtracker#6040]
![image](https://user-images.githubusercontent.com/1942409/60433086-371d9a80-9c04-11e9-8f59-84a545807934.png)

If the event starts
![image](https://user-images.githubusercontent.com/1942409/60433113-4b619780-9c04-11e9-9b2a-4129235c6099.png)
before the actual worldserver is available, it will result in spawning only 1 npc and the rest spawns after like half minute or so... its braindead because all of those npcs are handled in 1 trigger(23395)
![image](https://user-images.githubusercontent.com/1942409/60433215-8d8ad900-9c04-11e9-8421-fcca93d41655.png)

etc...

so basically it stops after spawning lietenant

and the trigger that handles the event appears dead after login...

**how it should work**: it either shouldnt spawn the trigger at all or spawn it fully (the event either plays fully or not at all)... i think game_events are being initialized before map spawns/spawned_by_script addon therefore causing part of the script to be played before disabling it

**Comment by Evolvee** [comment]
reproduction: 100%
just close and open the worldserver when the game_event is in progress(every 2 hours)
